### ISSUE TITLE
Test WINNF.FT.C.SCS.5 corrupted certificate using provided file on MockSAS not failled

### ISSUE THREAD

**Issue by gapelbaum** [Wireless-Innovation-Forum/Citizens-Broadband-Radio-Service-Device#121]
Hi,

I tested WINNF.FT.C.SCS.5 using the file sasharnesscertnotvalid.pem available on folder
"WINNF.FT.C.SCS.5_TLS failure certificate is corrupted" but TLS is established.

Someone tested it ?
Any advice.

Regards
Gustavo

**Comment by awaizkhan** [comment]
Hi Gustavo,
Please refer to the readme file for cert generation. All negative tests for TLS are mainly checking CBSD functionality to ensure in these -ve scenarios CBSD is able to take appropriate action. 

Please provide Wireshark trace for us to take a look?
-Awaiz

**Comment by gapelbaum** [comment]
Hi Awaiz,

Attached wireshark file, 192.168.4.125 is the Ip of MockSAS

[WINNF.FT.C.SCS.5.zip](https://github.com/Wireless-Innovation-Forum/Citizens-Broadband-Radio-Service-Device/files/1985218/WINNF.FT.C.SCS.5.zip)

Regards
Gustavo

**Comment by idanrazisr** [comment]
Hi Gustavo,
I tested again WINNF.FT.C.SCS.5 with the sample SAS corrupted certificate - and it is rejected properly as being corrupted.
I tested both using Airspan Domain Proxy and CURL utility and got the correct result - both rejected the TLS session by sending a TLS Alert message and and identified "certificate signature cannot be verified" or "decrypt error".
So your CBSD UUT failed to recognize this condition and wrongly continued to establish the TLS connection. I also noted that there are many TCP retrasnmissions from your SAS Harness machine - not sure why.
